src/styles/table.css

```diff
@@ -0,0 +1,25 @@
+/* table */
+
+table.ui.table {
+  border: none;
+}
+
+/** header **/
+
+table.ui.table thead tr th {
+  color: var(--primary-pms-539-900);
+  line-height: var(--line-height);
+  background-color: var(--shade-white);
+  border: none;
+  padding: var(--spacing-2);
+}
+
+/** cell **/
+
+table.ui.table tbody tr td {
+  color: var(--primary-pms-539-900);
+  line-height: var(--line-height);
+  font-weight: 400;
```

Comment:
I think we can remove `font-weight`, as that should be the default.

I'm not sure we need `line-height` either because this can be the same font size and line height as all of the over text on the page.

We may want to introduce a `site.css` stylesheet that styles text globally under the `body` selector.

Reply:
I am not sure if it's the same, tbh. I just removed it for now, and if we can find out if it is, we can create the site.css file and add the style property there.